### PR TITLE
Fix manage user menu alignment

### LINKED ISSUE #6207: User manage button menu gets misaligned - 2.2
When private chat messages are added to the userlist the dropdown menu for the user manage button becomes misaligned.

image

### PR DESCRIPTION
**Pull request by KDSBrowne**
Closes #6207

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/dropdown/styles.scss [hunk 1]
```diff
@@ -205,7 +205,7 @@
 
   &:after,
   &:before {
-    left: calc(var(--dropdown-caret-width) / 2);
+    left: calc(var(--dropdown-caret-width));
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/dropdown/styles.scss]: Line 197 for *right* caret is unchanged. Should we not keep them consistent?

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss [hunk 2]
```diff
@@ -1,43 +1,41 @@
...
+  --user-manage-menu-width : 10rem;
 }
 
 .dropdown {
-  position: static;
+  position: absolute;
+  right: 0px;
 }
-
+
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss]: please make sure to run `eslint --fix` , re-stage and then commit

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss [hunk 3]
```diff
@@ -1,43 +1,41 @@
...
-  --dropdown-offset-right: -0.2rem;
+  --user-manage-menu-top : -0.5rem;
+  --user-manage-menu-right : -0.75rem;
+  --user-manage-menu-width : 10rem;
 }
 
 .dropdown {
-  position: static;
+  position: absolute;
+  right: 0px;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss]: `px` can be omitted

File: bigbluebutton-html5/imports/ui/components/dropdown/styles.scss (hunk 1 above)
Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/components/dropdown/styles.scss]: If we change Line 197 to match this, the options dropdown gets misaligned.

image

Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/dropdown/styles.scss]: has some reason to call a `calc` function without a calculation?

**Pull request merged**